extensions-core/s3-extensions/src/main/java/org/apache/druid/storage/s3/S3TaskLogs.java

```diff
@@ -146,7 +147,18 @@ private void pushTaskFile(final File logFile, String taskKey) throws IOException
     }
   }
 
-  private String getTaskLogKey(String taskid, String filename)
+  private void uploadFileIfPossible(String bucket, String key, File file)
```

Comment:
I think you forgot to remove this method after consolidating it into S3Utils

Reply:
Removed.